hbase-shell/src/main/ruby/shell/commands/unassign.rb

```diff
@@ -22,8 +22,10 @@ module Commands
     class Unassign < Command
       def help
         <<-EOF
-Unassign a region. Unassign will close region in current location and then
-reopen it again.  Pass 'true' to force the unassignment ('force' will clear
+Unassign a region. It could be executed only when region in expected state(OPEN).
+In addition, you can use "unassigns" which supported by hbck2 to skip the state check.
+See https://github.com/apache/hbase-operator-tools/blob/master/hbase-hbck2/README.md for more info.
```

Comment:
I think we can write this at the end similar to above `assign.rb` (above Examples)

Reply:
And also, we can keep link in bracket like `(For more info on HBCK2: https://github.com/apache/hbase-operator-tools/blob/master/hbase-hbck2/README.md)`

Reply:
Make sense.
BTW, this part seems not correct too, since we do nothing for the "force" param in server side now.
```
Pass 'true' to force the unassignment ('force' will clear all in-memory state in
master before the reassign. If results in double assignment use hbck -fix to resolve.
To be used by experts).
```

Reply:
The semantic of unassign was changed, and not need param any more, maybe it is better to fix it in a separate Jira.